controllers/getWeather.js

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
Initially trying to go with this node module openweather-apis .. the 5 day forecast is not currently working, trying to resolve with module's author. may have to scrap code